.travis.yml

```diff
@@ -9,4 +9,6 @@ install:
 
 script:
   - npm test
-  - ./node_modules/sw-testing-helpers/project/publish-docs.sh master
+  - if [[ "$TRAVIS_BRANCH" = "master" && "$TRAVIS_OS_NAME" = "linux" && "$TRAVIS_PULL_REQUEST" = "false" ]]; then
```

Comment:
This is largely for security more than anything. I don't think it would happen, but at the same time, people do weird and wonderful things with git, let alone what Travis could do.